### ISSUE TITLE
Minifier breaks on a unitless media query

### ISSUE THREAD

**Issue by louh** [css/csso#336]
Tested on https://css.github.io/csso/csso.html

If I have this minimal test case (valid CSS):

```css
@media only screen and (-webkit-min-device-pixel-ratio: 2) {
    .test {
        color: white;
    }
}
```

I get this error:

```
Cannot read property 'prev' of undefined
```

I can add an arbitrary letter after the number (a "unit") to fix the output:

```css
@media only screen and (-webkit-min-device-pixel-ratio: 2a) {
    .test {
        color: white;
    }
}
```

But this would be invalid CSS.

**Comment by lahmatiy** [comment]
@louh Hi, issue was fixed in [3.1.1](https://github.com/css/csso/releases/tag/v3.1.1).
I forgot to update the web interface. Try it now.

**Comment by louh** [comment]
Works perfectly, thank you :)
